src/components/navigation/footer-nav.tsx

```diff
@@ -9,6 +10,7 @@ import { FooterNavigation } from '@/data-types'
 const iconMapping = {
   newsletter: faEnvelope,
   github: faGithubSquare,
+  job: faHouseLaptop,
```

Comment:
Where are they visible?

Reply:
On the "normal" footer e.g. on https://frontend-git-footer-changes-serlo.vercel.app/1555